### PR TITLE
Separate the basic translation from the specific (default/friendly)

### PR DESCRIPTION
**Pull request by tnonnenm**
Basis file (/lang/<locale>.json) contains all items that were identical in default & friendly.
The latter files contain only those values that are different (e.g warrior, battle, ...).

I tested this for locales en & de. If you like this idea I can do the separation also for the rest of the languages and attach them to this PR, too.

Please let me know what you think.

### CODE REVIEW

**Comment by StevenWeathers** [comment]
I've been planning to do this actually since a majority of the strings are common between the default and friendly experiences.  If you finish this out I will accept the PR.

File: frontend/src/i18n.js [hunk 1]
```diff
@@ -32,22 +33,36 @@ function setupI18n(options = {}) {
...
             addMessages(locale_, messages)
+        })
+
+        const messageFileUrlAdd = MESSAGE_FILE_URL_ADD.replace(
+            '{locale}',
+            locale_,
+        )
+
+        // Download additional translation file (default/friendly) for given locale/language
+        loadJson(messageFileUrlAdd).then(messages => {
```
Comment (StevenWeathers) [inline frontend/src/i18n.js:55]: So these are promises which means you will have to nest this additional loadJson call into the first one and move the isDownloading.set(false) into this one. e.g.

```
loadJson().then(messages => { loadJson().then(m => { isDownloading.set(false) }) })
```

Comment (tnonnenm) [reply frontend/src/i18n.js]: Done.

Comment (StevenWeathers) [reply frontend/src/i18n.js]: Thanks!

I'll pull this down and test today, appreciate all your hard work ;)

**Comment by tnonnenm** [comment]
> So this change would break the active status of the item, will have to add a label to each object that can be translated but keep the english name for the active status to work correctly

Sorry, missed that usage of the name.
Reverted the names and defined a label per entry used for rendering.

**Pull request merged**